### PR TITLE
Sweetalerts issue 792

### PR DESCRIPTION
**Pull request by ramavavi**
This PR refers to the issue: https://github.com/episphere/connect/issues/792

Code changes in this PR are:

-  Removed swal functions in the code: 

1. Participant is checked out
2. Collection ID(s) already exist for this participant
3. Confirm ID is correct for participant: ${firstName
4. Collection specimen data has been saved
5. Please Select 'Shipping Location
6. Please select Box(es) to review and ship
7. Tracking Ids do not match in one of the boxes.
8. Tracking input saved
9. Participant is checked in

- Remove extra comments and console.log statements in the code.

### CODE REVIEW

File: src/events.js [hunk 1]
```diff
@@ -558,95 +558,77 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
...
-            
             checkOutParticipant(data);
-
-            await swal({
-                title: "Success",
-                icon: "success",
-                text: `Participant is checked out.`,
-            });
+            showTimedNotifications({ title: 'Success', body: 'Participant is checked out.' });
+            console.log('checkOutFlag', checkOutFlag);
```
Comment (bransteitterbr) [inline src/events.js:564]: Left over console.log

Comment (ramavavi) [reply src/events.js]: Removed the log statement.

**Review by bransteitterbr — APPROVED** [review]

**Review by jhflorey — APPROVED** [review]

File: src/events.js [hunk 2]
```diff
@@ -543,95 +545,76 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
...
-
+        } else {
             const visitConcept = document.getElementById('visit-select').value;
-            
             for(const visit of visitType) {
-                if(data['331584571'] && data['331584571'][visit.concept]) {
-                    const visitTime = new Date(data['331584571'][visit.concept]['840048338']);
-                    const now = new Date();
-                    
+                if(data[conceptIds.collection.selectedVisit] && data[conceptIds.collection.selectedVisit][visit.concept]) {
```
Comment (Gbarra9) [inline src/events.js:555]:
```suggestion
                if (data[conceptIds.collection.selectedVisit] && data[conceptIds.collection.selectedVisit][visit.concept]) {
```

**Review by Gbarra9 — APPROVED** [review]

File: src/events.js [hunk 3]
```diff
@@ -543,95 +545,76 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
...
+const handleCheckInModal = async (data, visitConcept, query) => {
+    await checkInParticipant(data, visitConcept);
+
+    const checkInMessage = {
+        title: "Success",
+        body: "Participant is checked in.",
+        continueButtonText: "Continue to Specimen Link",
+    };
+
+    const checkInOnCancel = () => {  };
```
[the remark this answers is not in the document]
Comment (ramavavi) [reply src/events.js:603]: No, this is an empty function.

File: src/shared.js [hunk 4]
```diff
@@ -3287,3 +3288,51 @@ export const validateSpecimenAndParticipantResponse = (specimenData, participant
...
+        </div>
+      </div>
+    `;
+  
+    document.body.appendChild(modalContainer);
+    modalContainer.appendChild(modalContent);
+    modalContainer.classList.add('show');
+    modalContainer.style.display = 'block';
+
+    const closeBtn = document.getElementById('closeBtn');
```
[the remark this answers is not in the document]
Comment (ramavavi) [reply src/shared.js:3328]: Thanks for the suggestion, I have added the code for the null or undefined check.

**Review by JoeArmani — APPROVED** [review]
looks good

**Pull request merged**